src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/SessionLocationValidatorTest.kt

```diff
@@ -17,15 +17,12 @@ import uk.gov.justice.digital.hmpps.visitscheduler.model.entity.session.SessionT
 import uk.gov.justice.digital.hmpps.visitscheduler.model.entity.session.location.PermittedSessionLocation
 import uk.gov.justice.digital.hmpps.visitscheduler.model.entity.session.location.SessionLocationGroup
 import uk.gov.justice.digital.hmpps.visitscheduler.service.PrisonerService
+import uk.gov.justice.digital.hmpps.visitscheduler.utils.validators.SessionLocationValidator
 import java.time.LocalDate
 
 @ExtendWith(MockitoExtension::class)
-class PrisonerSessionValidatorTest {
-  private val prisonerLevelMatcher = PrisonerLevelMatcher()
-  private val prisonerCategoryMatcher = PrisonerCategoryMatcher()
-  private val prisonerIncentiveLevelMatcher = PrisonerIncentiveLevelMatcher()
-  private val prisonerSessionValidator = PrisonerSessionValidator(prisonerLevelMatcher, prisonerCategoryMatcher, prisonerIncentiveLevelMatcher)
-
+class SessionLocationValidatorTest {
```

Comment:
renamed this class as it was only doing Location tests.

Reply:
So why have you got a separation for your tests on location but not cat and incentive

Reply:
this was an existing test which has been renamed and refactored. Will add category and incentive tests.

Reply:
Added 2 new tests for category and incentives.